packages/London-Boroughs/package.json

```diff
@@ -0,0 +1,23 @@
+{
+	"name": "@svg-maps/London-Boroughs",
```

Comment:
Could you rename the folder and the package to be consistent with the other maps please?
`packages/london.boroughs/` and `@svg-maps/london.boroughs`

Reply:
Could you fix the names please? It's `london.boroughs` (like `france.regions`, `pakistan.districts`...)